### PR TITLE
feat: support EVALS_SHELL env var for custom command

### PR DESCRIPTION
**Pull request by stiyyagura0901**
- when the orb is used to run a custom command, now orb accepts an `optional` param `shell`. It defaults to `/bin/bash`.
- it runs as `/bin/bash -C <custom_command>`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,10 +77,12 @@ _Some of the parameters are optional based on the eval platform being used._
 
 - **`cmd`**: Command to run the evaluation
 
-- **`eval_platform`**: Evaluation platform (e.g. `braintrust`, `langsmith` etc.; default: `braintrust`)
+- **`eval_platform`**: Evaluation platform (e.g. `braintrust`, `langsmith` or `custom`; default: `custom`)
```
Comment (stiyyagura0901) [inline README.md:80]: this change is unrelated to this PR. Missed on earlier changes.

File: src/examples/run_eval_with_custom_command.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ description: >
 usage:
   version: 2.1
   orbs:
-    evals: circleci/evals@0.0
+    evals: circleci/evals@1.0
```
Comment (stiyyagura0901) [inline src/examples/run_eval_with_custom_command.yml:6]: this change is unrelated to this PR. Missed on earlier changes.

**Review by ryan-circleci — APPROVED** [review]
